tests/service/test_standard_interfaces.py

```diff
@@ -108,8 +109,9 @@ async def introspect_subpath(path, expected_subnodes):
 
     await introspect_subpath("/", {"a"})
     await introspect_subpath("/a", {"test", "subpath"})
-    await introspect_subpath("/a/test", {"path1"})
+    await introspect_subpath("/a/test", {"path1", "path10"})
     await introspect_subpath("/a/test/path1", set())
```

Comment:
It looks like we need some more coverage for passing paths with a trailing slash since the new check above it doing `path and path[-1] != "/"`

Reply:
I think `/`is the only valid object path with a trailing slash. I adapted the check to only test `path != "/"`.